network-api/networkapi/templates/pages/base.html

```diff
@@ -110,8 +110,8 @@
         {% endif %}
 
         {% block donate_banner %}
-            {% if settings.donate_banner.SiteDonateBanner.active_donate_banner %}
-                {% include "fragments/donate_banner.html" with banner=settings.donate_banner.SiteDonateBanner.active_donate_banner.localized %}
+            {% if donate_banner %}
+                {% include "fragments/donate_banner.html" with banner=donate_banner.localized %}
```

Comment:
Instead of displaying the donate banner set in the site’s settings, we are now rendering the banner provided to the context by the `get_donate_banner()` method.

Reply:
Great point, @mmmavis!

Since setting a donate banner on the homepage already ensures the banner is displayed sitewide (or at least on the pages listed in the description), keeping the "Donate Banner" field in the side menu under "Settings" could be redundant and confusing for users.

With that in mind, I am opting to remove it from the site settings menu for now, and letting users know that we are now setting it on the Homepage. What do you think?

 cc'ing @robdivincenzo as well to get his thoughts on this.

Reply:
The "Donate Banner" option has been removed from the side menu -> settings for now. Going to mark this as resolved!